docs/installation/server-config/security.rst

```diff
@@ -132,28 +133,213 @@ Configuring Authorization of KSQL with Kafka ACLs
+
+    # Allow KSQL to read the input topics (including output-topic1):
+    bin/kafka-acls --authorizer-properties zookeeper.connect=localhost:2181 --add --allow-principal User:KSQL1 --allow-host 198.51.100.0 --allow-host 198.51.100.1 --allow-host 198.51.100.2 --operation Read --topic input-topic1 --topic input-topic2 --topic output-topic1
+
+    # Allow KSQL to write to the output topics:
+    bin/kafka-acls --authorizer-properties zookeeper.connect=localhost:2181 --add --allow-principal User:KSQL1 --allow-host 198.51.100.0 --allow-host 198.51.100.1 --allow-host 198.51.100.2 --operation Write --topic output-topic1 --topic output-topic2
+    # Or, if the output topics do not already exist, the 'create' operation is also required:
+    bin/kafka-acls --authorizer-properties zookeeper.connect=localhost:2181 --add --allow-principal User:KSQL1 --allow-host 198.51.100.0 --allow-host 198.51.100.1 --allow-host 198.51.100.2 --operation Create --operation Write --topic output-topic1 --topic output-topic2
+
+    # Allow KSQL to manage its own internal topics and consumer groups:
+    bin/kafka-acls --authorizer-properties zookeeper.connect=localhost:2181 --add --allow-principal User:KSQL1 --allow-host 198.51.100.0 --allow-host 198.51.100.1 --allow-host 198.51.100.2 --operation All --resource-pattern-type prefixed --topic _confluent-ksql-production_ --group _confluent-ksql-production_
+
+
+.. _config-security-ksql-acl-interactive_post_ak_2_0
+
+Interactive KSQL clusters
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:ref:`Interactive KSQL clusters <restrict-ksql-interactive>` accept SQL statements from users and hence may require access
+to a wide variety of input and output topics. Add ACLs to appropriate literal and prefixed resource patterns to allow KSQL
+access to the input and output topics, as required.
+
+Recommendations for securing interactive KSQL clusters:
+
+- To simplify ACL management, you should configure a default custom topic name prefix such as ``ksql-interactive-`` for your
+KSQL cluster via the ``ksql.output.topic.name.prefix`` :ref:`server configuration setting <set-ksql-server-properties>`.
+Unless a user defines an explicit topic name in a KSQL statement, KSQL will then always prefix the name of any automatically
+created output topics.
+- Add an ACL to allow ``ALL`` operations on ``TOPIC``s ``PREFIXED`` with the configured custom name prefix (in the example above: ``ksql-interactive-``).
+
+For example, given the following setup:
+
+- A 3-node KSQL cluster
+- Running on hosts with IPs 198.51.100.0, 198.51.100.1, 198.51.100.2
+- Authenticating with the Kafka cluster as a 'KSQL1' user.
+- With 'ksql.service.id' set to 'fraud_.
+- Where users should be able to run queries against any input topics prefixed with 'accounts-', 'orders-' and 'payments-'.
+- Where 'ksql.output.topic.name.prefix' is set to 'ksql-fraud-'
+- And users won't use explicit topic names.
```

Comment:
Can we clarify with a 1-liner what "explicit topic names" are?  It's not a formal term, and we haven't really used it.  I'd expect that many people will skip over the text and just read the example and this bullet list. ;-)

Perhaps:

----

- And users won't use explicit topic names, i.e. users will rely on KSQL auto-creating any required topics with auto-generated names.  (Note: If users want to use explicit topic names, then you must provide the necessary ACLs for these in addition to what's shown in the example below.)

----